src/gui/properties/downloadedpiecesbar.h

```diff
@@ -37,49 +37,63 @@
 #include <QBitArray>
 #include <QVector>
 
+class QHelpEvent;
+
+namespace BitTorrent
+{
+    class TorrentHandle;
+}
+
```

Comment:
And I was wondering from where these came from(I was looking at the 2nd commit diff only).
These don't belong in this commit.